environment.yml

```diff
@@ -26,12 +26,16 @@ dependencies:
 - bqplot
 - scikit-learn>=0.19.1
 - pip>=19.0.3
+- cython>=0.29.1
+- papermill>=0.15.0
+- black>=18.6b4
+- ipywebrtc 
+- lxml>=4.3.2
+- pre-commit>=1.14.4
+- pyyaml>=5.1.2
+- requests>=2.22.0
+- cython
```

Comment:
cython is twice, apart from that, this looks good

Reply:
Good catch! Let's use this .yaml file for now to unblock you in master. We will update very soon anyway when merging staging into master. Will remove the second cython now in staging.